pkg/kubectl/cmd/create.go

```diff
@@ -113,3 +117,24 @@ func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) err
 	}
 	return nil
 }
+
+func printObjectSpecificMessage(obj runtime.Object, out io.Writer) {
+	switch obj := obj.(type) {
+	case *api.Service:
+		if obj.Spec.Type == api.ServiceTypeLoadBalancer {
```

Comment:
Where do we document the overly-loose GCE firewall problem?  Should we stick it in a .md and link it here?